docs/dev_flags.md

```diff
@@ -0,0 +1,8 @@
+# Developer flags
+
+Set the following developer flags in your shell startup script of choice (e.g. `.bashrc`, `.bash_profile`, `.zshrc`):
+
+```bash
+# Enables local development of the Corpora app
+export CORPORA_LOCAL_DEV=true
+```
```

Comment:
Could also set the DEPLOYMENT_STAGE variable?

Reply:
i.e. default to `dev`? Sure, though I felt that one was developer controlled whereas this one is meant to be constant. Specifically its purpose is differentiating a development deployment of the app from and production deployment. Right now, it's just being used to connect to the RDS via `localhost:5433` (development deployment) or the live RDS endpoint (production deployment).

Reply:
That's fair! Can disregard the suggestion :)